docs/dev_guide/pr_review_procedure.rst

```diff
@@ -0,0 +1,81 @@
+This document describes the procedure that the people in the
+sunpy-maintainters group must follow when merging a pull request. If in
+any doubt about merging a pull request, the correct course of action is
+to contact the Lead Developer / Executive Director and ask.
+
+Fundamental Components of a Pull Request
+========================================
+
+Each pull request *must* meet the following criteria before it is
+considered for merge:
+
+-  The code must be PEP 8 compliant and meet the ill-defined SunPy
+   quality standards.
+-  The PR must contain a CHANGELOG entry if a sunpy-maintainter deems so.
+-  The test coverage **ABSOLUTELY** must not decrease, and for new
+   features should be at or very close to 100%.
+-  All code must be properly documented. Each function and each class
+   must have an associated doc string in the correct format.
+
+Review Process
+==============
+
+Before the ‘merge’ button is clicked the following criteria must be met:
+
+-  All the builds pass (currently Travis CI, Appveyor and Coveralls.
+-  At least two members (not the author of the PR) of the
+   sunpy-developers group have approved the PR. (Normally via the GitHub
+   `review`_ or by commenting on the PR with either a :+1:(``:+1:``)
+   emoticon or +1).
```

Comment:
I think we should remove the `+1` as now we have a `approve` button.

Reply:
I think too that using PR review infrastructure is a much more clear approach.

You already protected the master branch anyway, so at least one maintainer with commit rights has to give the approval.